src/markdown-pages/build-apps/build-visualization.mdx

```diff
@@ -0,0 +1,210 @@
+
+## Create a new visualization
+
+In this first set of steps you use the CLI to quickly generate the boilerplate for a new visualization.
+
+<Steps>
+
+  <Step>
+
+Ensure you're working with the latest version of the New Relic One CLI:
+
+```bash
+nr1 update
+```
+
+  </Step>
+
+  <Step>
+
+Start with a new Nerdpack. If you just installed the New Relic One CLI, then you have a new Nerdpack project directory (called something like, `my-awesome-nerdpack`). If you need to create one, run `nr1 create` and select the `nerdpack` option.
+
+```bash
+nr1 create --type nerdpack --name my-awesome-nerdpack
+```
+
+  </Step>
+
+  <Step>
+
+Navigate to the root of your Nerdpack project, and then run the `create` command.
+
+```bash
+cd my-awesome-nerdpack
+nr1 create --type visualization --name my-awesome-visualization
+```
+
+As a result, you have a new visualization directory matching the name you gave your visualization. It is located in the `visualizations` directory in the root of your nerdpack project.
+
+<img
+  src="../../images/build-an-app/visualization-boilerplate-files.png"
+  alt="Visualization files generated by CLI"
+/>
+
+The files created provide an example visualization – a radar chart populated by a basic NRQL query.
+
+The `nr1.json` file provides metadata including the `configuration` field that defines the prop input fields to be shown in the UI. These are the fields users will fill in to create an instance of the visualization. Supported input types are `boolean`, `string`, `number`, `json`, `nrql`, `namespace`, and `collection`. A `namespace` type groups a set of props together in the prop editing UI under a heading. A `collection` defines a set of fields that can be repeated and result in an array prop type. The `nrqlQueries` entry is an example of a collection type.
+
+The `index.js` file is where you define the React component that receives the props and renders the visualization. You can install and import any javascript libraries you find useful. The example component imports [recharts](https://recharts.org), a library that wraps [D3](https://d3js.org) submodules for easy to compose visualizations.
+
+If your visualization queries data over a time range and you want it to utilize the platform time range setting, you will need to utilize the `timeRange` value from [PlatformStateContext](/components/platform-state-context). The `timeRange` can be passed directly to the [NrqlQuery](/components/nrql-query) component. Setting the NrqlQuery component's `pollInterval` prop to `NrqlQuery.AUTO_POLL_INTERVAL` automatically handles calculating an appriate data polling interval.
+
+```js
+<PlatformStateContext.Consumer>
+  {({timeRange}) => (
+    <NrqlQuery
+      timeRange={timeRange}
+      pollInterval={NrqlQuery.AUTO_POLL_INTERVAL}
+    >
+    ...
+```
+
+  </Step>
```

Comment:
This section is all showing as `commented` code in Github, i'm wondering if that due to the line on 92 that I suggested to remove? Is this section rendering correctly locally?

Reply:
Interesting, it is rendering correctly locally.